packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -122,8 +122,12 @@ export class DownloadJobs {
             parameters += "?mode=record";
         }
 
+        if (!parms.binary && !parms.record && parms.encoding && parms.encoding.trim() != "") {
```

Comment:
`parms.encoding && parms.encoding.trim() != ""` could be simplified to `parms.encoding?.trim()` 😅